examples/plot_pp_study.py

```diff
@@ -0,0 +1,362 @@
+
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+"""
+================================
+Predictive Policing Case Study
+================================
+"""
+# %%
+#
+#
+# Contents
+# ========
+#
+# 1. Overview
+# 2. Background
+# 3. Fairlearn Results
+# 4. Problem area & Sociotechnical context
```

Comment:
I feel like the context is by far the most important. I'm not sure it even needs to be separate from background (?). The use of Fairlearn and demonstrating why that's not a solution to the problem is a great bonus, but the focus is elsewhere (at least in my mind).